### ISSUE TITLE
vault-id support

### ISSUE THREAD

**Issue by divansantana** [zellio/ansible-vault-mode#21]
```
 ansible-vault encrypt_string --encrypt-vault-id nonprod --vault-id nonprod@vault/vault-nonprod.txt '117abc0e519d7830b896730315e25e9fa4'
!vault |
          $ANSIBLE_VAULT;1.2;AES256;nonprod
          36636237343538623133323330303364373639386338646639346634366262343333366265363235
          3439626436363631326333623361346333613462343434360a313566656535633534343736393166
          62343238656466653065323835366532653931613465366665303332363637653362663765326463
          6661353833666231370a626130386364323637653261393961366135363262333835663233343461
          35663066303064373231393034363361656561333762363964356466653130646639383234316134
          3138663732656535336537303062666364303832333936346135
Encryption successful
```

as per above, one can encrypt using a passwordfile and vault-id.

Is it possible to support vault-id? AFAIK the code doesn't currently support that?

**Comment by zellio** [comment]
I don't use Ansible professionally anymore so I may or may not get to this issue. Happy to accept and review pull requests.

**Comment by zellio** [comment]
Turns out I had some time. As I don't really use Ansible much anymore would you be willing to test the feature branch I just pushed up?

Vault Id Support: [feature/vault-id-handling](https://github.com/zellio/ansible-vault-mode/tree/feature/vault-id-handling)

**Comment by zellio** [comment]
This has been resolved in release v0.5.0
